QuizApp/QuizApp/Network/NetworkClient.swift

```diff
@@ -0,0 +1,99 @@
+import Foundation
+
+class NetworkClient {
+
+    private let baseUrl: String
+    private let secureStorage: SecureStorage
+
+    init(secureStorage: SecureStorage, baseUrl: String) {
+        self.secureStorage = secureStorage
+        self.baseUrl = baseUrl
+    }
+
+    func get(path: String) async throws {
+        guard let url = URL(string: "\(baseUrl)\(path)") else {
+            throw RequestError.invalidURLError
+        }
+
+        let accessToken = secureStorage.fetchAccessToken() ?? ""
```

Comment:
`secureStorage.fetchAccessToken()` - functions that do not have arguments and return a value should be computed properties.